idb/postgres/internal/writer/writer_test.go

```diff
@@ -1565,6 +1565,102 @@ func TestWriterAddBlock0(t *testing.T) {
 		assert.Equal(t, expected, accounts)
 	}
 }
+func getNameAndAccountPointer(t *testing.T, value *string, fullKey string, accts map[basics.Address]*ledgercore.AccountData) (basics.Address, string, *ledgercore.AccountData) {
+	require.NotNil(t, value, "cannot handle a nil value for box stats modification")
+	appIdx, name, err := logic.SplitBoxKey(fullKey)
+	account := appIdx.Address()
+	require.NoError(t, err)
+	acctData, ok := accts[account]
+	if !ok {
+		acctData = &ledgercore.AccountData{
+			AccountBaseData: ledgercore.AccountBaseData{},
+		}
+		accts[account] = acctData
+	}
+	return account, name, acctData
+}
+
+func addBoxInfoToStats(t *testing.T, fullKey string, value *string,
+	accts map[basics.Address]*ledgercore.AccountData, boxTotals map[basics.Address]basics.AccountData) {
+	addr, name, acctData := getNameAndAccountPointer(t, value, fullKey, accts)
+
+	acctData.TotalBoxes++
+	acctData.TotalBoxBytes += uint64(len(name) + len(*value))
+
+	boxTotals[addr] = basics.AccountData{
+		TotalBoxes:    acctData.TotalBoxes,
+		TotalBoxBytes: acctData.TotalBoxBytes,
+	}
+}
+
+func subtractBoxInfoToStats(t *testing.T, fullKey string, value *string,
+	accts map[basics.Address]*ledgercore.AccountData, boxTotals map[basics.Address]basics.AccountData) {
+	addr, name, acctData := getNameAndAccountPointer(t, value, fullKey, accts)
+
+	prevBoxBytes := uint64(len(name) + len(*value))
+	require.GreaterOrEqual(t, acctData.TotalBoxes, uint64(0))
+	require.GreaterOrEqual(t, acctData.TotalBoxBytes, prevBoxBytes)
+
+	acctData.TotalBoxes--
+	acctData.TotalBoxBytes -= prevBoxBytes
+
+	boxTotals[addr] = basics.AccountData{
+		TotalBoxes:    acctData.TotalBoxes,
+		TotalBoxBytes: acctData.TotalBoxBytes,
+	}
+}
+
+// buildAccountDeltasFromKvsAndMods simulates keeping track of the evolution of the box statistics
+func buildAccountDeltasFromKvsAndMods(t *testing.T, kvOriginals, kvMods map[string]*string) (
```

Comment:
@tzaffi Tangentially related to the line:  I didn't research the history, but there appears to be a duplicate + unused `BuildAccountDeltasFromKvsAndMods` function defined in `util/test/box_testutil.go`.  

From a cursory review, it looks like `util/test/box_testutil.go` is wholesale unused.  Can it be removed?

Reply:
Oops. This is a good catch and concerning. I've been through several test refactorings, but I'm surprised this ended up unused. The basic functionality ought to be tested. I will now investigate to see if I duplicated this elsewhere or need to rewire it to an actual test.

Reply:
After further investigation, all the tests are in place and `util/test/box_testutil.go` is unneeded (probably the result of a poorly resolved merge conflict). I will remove this unnecessary file even as it reduces my line count bragging rights by almost 150.